client/src/slices/snackbar/slice.ts

```diff
@@ -31,38 +28,7 @@ const snackbarSlice = createSlice({
       if (state.snacks?.[action.payload]) delete state.snacks[action.payload];
     },
   },
-  extraReducers: builder => {
-    builder.addMatcher(
-      action => action.type.endsWith('/rejected'),
-      (state, action: UnknownAsyncThunkRejectedAction) => {
-        const error = action.payload as AxiosError<ApiError>;
-        const defaultError = {
-          data: { message: 'Network problem probably, if not, contact dev' },
-        };
-        if (error.isAxiosError) {
-          const message = (error.response ?? defaultError).data.message;
-          state.snacks[message] = {
-            dismissed: false,
-            message: message,
-            key: message,
-            options: {
-              variant: 'error',
-            },
-          };
-        } else {
-          const key = error.message;
-          state.snacks[key] = {
-            dismissed: false,
-            message: 'contact the developer, error: ' + error.message,
-            key: key,
-            options: {
-              variant: 'error',
-            },
-          };
-        }
-      },
-    );
-  },
+  extraReducers: builder => extraReducers(builder),
```

Comment:
bring this back in, because the extra reducers for this slice wont increase nor the core reducers

the slice is good enough to have all the reducers in them as you'd have a big "reducer.ts" when using plain redux